qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -823,23 +823,23 @@ test_status_in_pool_dir()
     local description_pattern="$5"
     local service_pattern="$6"
 
-    local status_log=${TEMPDIR}/$(mkfname ${cluster}-${pool}-${image}.mirror_status)
-    CEPH_ARGS='' rbd --cluster ${cluster} mirror image status ${pool}/${image} |
-        tee ${status_log} >&2
-    grep "^  state: .*${state_pattern}" ${status_log} || return 1
-    grep "^  description: .*${description_pattern}" ${status_log} || return 1
+    local status=$(CEPH_ARGS='' rbd --cluster ${cluster} mirror image status \
+                     ${pool}/${image})
```

Comment:
It looks the output of the command was making it to the log before.  Should it be echoed before running any of the greps?